src/sass/static/pages.scss

```diff
@@ -694,7 +694,7 @@ body #not_authenticated_financial ul.checked > li {
 
 .accounts-content {
     .table-container {
-        overflow-x: hidden;
+        overflow-x: visible;
```

Comment:
so do we need this style at all?

Reply:
@negarn Due to having the responsive table we no longer need horizontal scroll in these tables, and when I set it to hidden it causes the currency dropdown to be hidden, but visible works fine